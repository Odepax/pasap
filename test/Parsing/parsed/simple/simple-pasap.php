<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<article>
			<h1>Some Pasap Custom Tags</h1>
			<one/>
			<two/>
			<three/>
		</article>
	</body>
</html>
