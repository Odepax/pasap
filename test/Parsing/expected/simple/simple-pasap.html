<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<article>
			<h1>Some Pasap Custom Tags</h1>
			<h5>One</h5>
			<h5>Two</h5>
			<h5>Three</h5>
		</article>
	</body>
</html>
